server/chat/backend/agent/utils/persistence/context_manager.py

```
@@ -208,4 +211,6 @@ async def flush_session(cls, session_id: str) -> bool:
     def cleanup(cls):
```

Ruff (0.15.10)

[warning] Missing return type annotation for classmethod `cleanup`

Add return type annotation: `None`

(ANN206)